composer.json

```diff
@@ -50,7 +52,6 @@
 				"Apply PKP ADODB patches": "lib/adodb.diff"
 			},
 			"laravel/framework": {
-				"Inhibit __ Laravel helper": "lib/laravel-helper-4017.diff",
```

Comment:
Just curious about this -- the `gettext/translator` package appears to provide its own `__` function, as does `laravel/framework`. How do these two avoid collision?

Reply:
Sure, the `__()` function from `gettext/translator` isn't included by default.